code/src/index.css

```diff
@@ -1,13 +1,26 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  -moz-osx-font-smoothing: grayscale; 
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+.todo-item {
+  margin-top: 20px;
+}
+
+.complete {
+  text-decoration: line-through;
+  opacity: 0.7;
+}
```

Comment:
Think about how this part can be done with styled components instead as the rest of the styling. :) Let me know if you want help to think about it. :)